src/yaml/mapping.cr

```diff
@@ -29,16 +29,15 @@ module YAML
   #
   # ```
   # employee = Employee.from_yaml("title: Manager\nname: John\nage: 30")
-  # employee.age # => undefined method 'age'.
+  # employee.age # undefined method 'age'. (compile error)
   #
-  # employee = Employee.from_yaml("title: Manager")
-  # # => ParseException: missing yaml attribute: name
+  # Employee.from_yaml("title: Manager") # raises YAML::ParseException
   # ```
   #
   # You can also define attributes for each property.
   #
   # ```
-  # class Employee
+  # class Employer
```

Comment:
@Sija 
This is intended. And this issue is a bit complex. In this src, two classes are defined as same name `Employee`. Those properties are similar but not same about `name`.

And my automatic CI app generates 1 spec( `it`) for 1 source as following. So, it fails when class name is reused.
https://github.com/maiha/crystal-examples/blob/master/gen/spec/yaml/mapping_spec.cr

Thus, we have two choices.
1. Use another class name. (I choose this)
2. Keep `Employee` but skips to test. (need to skip at least currently)

Thought?